### DISCUSSION TITLE
How to get rounded corner and transparency for Overview widget?

### DISCUSSION THREAD

**Discussion by SherLock707** [end-4/dots-hyprland#371] [Q&A] [answered]
Right now it looks like this on my Hyprland:

![image](https://github.com/end-4/dots-hyprland/assets/26952545/19b03061-4028-4300-961f-563bb9a6728f)

How can I add bit of transparency and  rounded corner to the widget.

I'm completely noob at ags scripts.

**Comment by end-4** [comment]
please send output of 
```
sass ~/.config/ags/scss/main.scss ~/.cache/ags/user/generated/style.css
```

**Reply by SherLock707** [reply, marked as answer]
`sass` was missing, installed it. thanks. fixed.
